internals/secrethub/generate.go

```diff
@@ -254,6 +286,10 @@ type intValue struct {
 	v *int
 }
 
+func (iv *intValue) Type() string {
```

Comment:
We should also drop the `intValue` type and just use the cobra int flag type instead. Or does this type have any other functionality attached to it?

Reply:
I also saw this and attempted its removal, however this type is involved in a few other functions, such as this:
```
func (cmd *GenerateSecretCommand) length() (int, error) {
	if cmd.lengthArg.IsSet() && cmd.lengthFlag.IsSet() {
		return 0, ErrCannotUseLengthArgAndFlag
	}
	if cmd.lengthFlag.IsSet() {
		return cmd.lengthFlag.Get(), nil
	}
	if cmd.lengthArg.IsSet() {
		return cmd.lengthArg.Get(), nil
	}
	return defaultLength, nil
}
```

Reply:
We decided to keep this type for now as it is only used in one place. We will look for options to remove it if possible, after the major design changes we agreed on are implemented.

Reply:
There is a way to simplify the struct and methods that come with the structure. Howerver, there is no way to properly remove it.
For arguments, we encapsulate each type in a separate struct so that we can easily transform the received string into the proper type (e.g. from string to URI). In this way, we keep the argument paersing part simple and compact by avoiding to say in each command how to parse the arguments. 
Let me know if you have any curiosities or ideas that may come up with this. In my opinion, We won't fully eliminate it, rather simplify it and put it in the `env` Go file from the `cli` package.

Reply:
Based on the discussion with SImon, We will keep the `intValue` struct here for backwards-compatibility purposes.